cmd/goose/driver_mysql.go

```diff
@@ -64,6 +64,21 @@ func registerTLSConfig(pemfile string) error {
 	if ok := rootCertPool.AppendCertsFromPEM(pem); !ok {
 		return fmt.Errorf("failed to append PEM: %q", pemfile)
 	}
+
+	if sslcert != "" && sslkey != "" {
```

Comment:
Small nit. What are your thoughts about simplifying this chunk of code?

```go
tlsConfig := &tls.Config{
	RootCAs: rootCertPool,
}
if sslcert != "" && sslkey != "" {
	cert, err := tls.LoadX509KeyPair(sslcert, sslkey)
	if err != nil {
		return fmt.Errorf("failed to load x509 keypair: %w", err)
	}
	tlsConfig.Certificates = append(tlsConfig.Certificates, cert)
}
return mysql.RegisterTLSConfig(tlsConfigKey, tlsConfig)
```

Other then this, looks good to me. Thanks for your contribution.

Reply:
Done and tested in local.